logstash.advisories.yaml

```diff
@@ -25,6 +25,10 @@ advisories:
         type: pending-upstream-fix
         data:
           note: Logstash bundles an old version of jruby v3.2.5 which installs a vulnerable version of rexml. Upstream jruby should fix this vulnerability for this version as it updates its default gems on some next release.
+      - timestamp: 2024-08-20T12:40:53Z
```

Comment:
I'd use the latest event here, as a reference. Perhaps we could explain why we have to vulnerable versions in this package v3.3.2 and v3.3.5

Reply:
> I'd use the latest event here, as a reference. 

@hectorj2f I don't follow. This is a new event with further information.

> Perhaps we could explain why we have to vulnerable versions in this package v3.3.2 and v3.3.5

v3.3.2 is a the current version being bundled/vendored as can be seen in a recent scan of the image.

```
➜  grype cgr.dev/chainguard-private/logstash-oss-with-opensearch-output-plugin
 ✔ Vulnerability DB                [no update available]  
 ✔ Loaded image                                                                                                                                                                                                                                                                                                                                             cgr.dev/chainguard-private/logstash-oss-with-opensearch-output-plugin:latest
 ✔ Parsed image                                                                                                                                                                                                                                                                                                                                                  sha256:79ceeb5ab62ca9eb2feb9efa23e903ffe65d28a294ebb29a35809afd6c5c8bc3
 ✔ Cataloged contents                                                                                                                                                                                                                                                                                                                                                   89e5b3295f2c294b66167bdebd65477245f3dfb5640cbd49f58c81356114c322
   ├── ✔ Packages                        [1,823 packages]  
   ├── ✔ File digests                    [19,125 files]  
   ├── ✔ File metadata                   [19,125 locations]  
   └── ✔ Executables                     [122 executables]  
 ✔ Scanned for vulnerabilities     [6 vulnerability matches]  
   ├── by severity: 0 critical, 0 high, 6 medium, 0 low, 0 negligible
   └── by status:   6 fixed, 0 not-fixed, 0 ignored 
NAME   INSTALLED  FIXED-IN  TYPE  VULNERABILITY        SEVERITY 
fugit  1.11.0     1.11.1    gem   GHSA-2m96-52r3-2f3g  Medium    
rexml  3.3.2      3.3.3     gem   GHSA-r55c-59qm-vjw6  Medium    
rexml  3.3.2      3.3.3     gem   GHSA-5866-49gr-22v4  Medium

```

v3.2.5 was the version bundled at the time this github issue https://github.com/chainguard-dev/CVE-Dashboard/issues/9917 was created. v3.2.5 does not appear in any scan reports now.

Reply:
Okay, I was checking if we wanted to show the latest event of all the events from our advisory. We are good with what you added.